pages/your_tasks/metadata_management.md

```diff
@@ -51,18 +51,19 @@ Due to the large variety of experiments, techniques and collaborative studies th
 
 ### Solutions
 * There are many appropriate tools for data documentation during the project. Check with your institute to know what is offered.
-  * Electronic Lab Notebooks (ELNs) are usually better for more disparate and unstructured information that requires flexibility. Researchers can use ELN in a personalized way and adapt it to document their every-day work.
+  * Electronic Lab Notebooks (ELNs), are usually better for more disparate and unstructured information that requires flexibility. Researchers can use an ELN (such as {% tool "elabftw" %}) in a personalized way and adapt it to document their every-day work.
```

Comment:
```suggestion
  * Electronic Lab Notebooks (ELNs) are usually better for more disparate and unstructured information that requires flexibility. Researchers can use an ELN (such as {% tool "elabftw" %}) in a personalised way and adapt it to document their everyday work.
```

Reply:
The original purpose of this commit was to add the reference to the elabFTW somewhere in the main text so it can be added to the tools (and eventually later referred to in the Portugal resources page). The suggestion you've made removes the tool from the text altogether.
@bianchini88 Do you have any other suggestions on where it may go or you're suggesting that this resource has no place in the main text at all?